Source/control.cpp

```diff
@@ -648,6 +648,21 @@ bool IsChatAvailable()
 #endif
 }
 
+namespace {

```

Comment:
`control.cpp` already has a anonymous namespace.
Should we move `GetHotkeyName` inside of the existing one?

Reply:
You're right.